var/ramble/repos/builtin/applications/quantum-espresso/application.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import os
+from ramble.appkit import *
+from ramble.expander import Expander
+
+
+class QuantumEspresso(SpackApplication):
+    '''Define Quantum-Espresso application.
+    https://www.quantum-espresso.org/
+    '''
+    name = 'quantum-espresso'
+
+    tags = ['electronic-structure', 'materials', 'dft', 'density-functional-theory', 'plane-waves', 'pseudopotentials']
+
+    default_compiler('gcc12', spack_spec='gcc@12.2.0', compiler_spec='gcc@12.2.0')
+
+    software_spec('impi2021p8', spack_spec='intel-oneapi-mpi@2021.8.0', compiler='gcc12')
+
+    software_spec('quantum-espresso', spack_spec='quantum-espresso@7.1', compiler='gcc12')
+
+    required_package('quantum-espresso')
+
+    input_file('AUSURF112', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/AUSURF112.tgz',
+               sha256='2b71c27801f8abbde05f48297b56b3194be0f83be50c50433f44bb886cac5117',
+               description='Input file for AUSURF112 benchmark')
+
+    input_file('CNT10POR8', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/CNT10POR8.tgz',
+               sha256='65e36acd332c1511cfa9cbdab0170a867b3144c98103c5bff3c5621ef795c00a',
+               description='Input file for CNT10POR8 benchmark')
+
+    input_file('GRIR443', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/GRIR443.tgz',
+               sha256='18016da632cde8624faeb07f3de04151a02f3b885aa4cbc111d6b912215403f8',
+               description='Input file for GRIR443 benchmark')
+
+    input_file('GRIR686', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/GRIR686.tgz',
+               sha256='4ade7691f88dd322b41abecbc08cc55437f9d376142e62b615317598abe498d7',
+               description='Input file for GRIR686 benchmark')
+
+    input_file('PSIWAT', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/PSIWAT.tgz',
+               sha256='698fbdac9f307b9ebc77e70e52574d67adb0f21bfd6874559d633471664b77d6',
+               description='Input file for PSIWAT benchmark')
+
+    # Set of small benchmarks to build a workload from.
+    # input_file('pwscf-small', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/pwscf-small-benchmark.tar.gz',
```

Comment:
If you want to keep this, shall we just uncomment it? (or remove it entirely?)

Reply:
I can remove it as I'm not planning on adding support for it yet. But I had wanted to leave it in here because it's a set of smaller benchmarks that one can run. The problem with it is you need to generate input files (they have a bash script in the repo that I had considered porting to python, but it seemed unnecessary).